usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -13,6 +13,7 @@
      ==========
 ****************************************************************************/
 
+#define _GNU_SOURCE
```

Comment:
why do we need this ? maybe add a comment
may it make sense to add this to the global compile options ?

Reply:
secure_getenv is only available if _GNU_SOURCE is defined.  Do we need it everywhere?  I added a comment.